### PR TITLE
fix schemas not being exported for lookup in other packages

### PR DESCRIPTION
**Pull request by Arnatious**
Fixes issue preventing schemas from loading in ros2nodl. Also fix tests not failing CI

Signed-off-by: Ted Kern <ted.kern@canonical.com>

### CODE REVIEW

File: nodl_python/nodl/schemas/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+# importlib does not support subdirectories, so schemas must be an (empty) module.
+# Will change in python 3.9
```
Comment (kyrofa) [inline nodl_python/nodl/schemas/__init__.py:14]: Package, not module, and not exactly empty.

```suggestion
# importlib.resources only supports importing from packages (fixed in python 3.9)
```

File: nodl_python/nodl/schemas/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Canonical Ltd.
```
Comment (kyrofa) [inline nodl_python/nodl/schemas/__init__.py:1]: If this is suddenly importable, we need to remove it from public API. Easiest way to do that is to make it `_schemas`.

Comment (Arnatious) [reply nodl_python/nodl/schemas/__init__.py]: fixed in ac62c82

File: nodl_python/nodl/schemas/__init__.py (hunk 1 above)
Comment (Arnatious) [reply nodl_python/nodl/schemas/__init__.py]: fixed in ac62c82

**Pull request merged**